libkbfs/bserver_remote.go

```diff
@@ -182,6 +182,26 @@ func (b *BlockServerRemote) ShouldRetryOnConnect(err error) bool {
 	return !inputCanceled
 }
 
+func makeBlockIDCombo(id BlockID, context BlockContext) keybase1.BlockIdCombo {
+	// ChargedTo is somewhat confusing when this BlockIdCombo is
+	// used in a BlockReference -- it just refers to the original
+	// creator of the block, i.e. the original user charged for
```

Comment:
The reason it was called `ChargedTo` was because when we add groups, blocks can be charged to the groups, but not necessarily to the original user that created it.  We didn't have that some foresight with the `Creator` and `Writer` fields, so I'm not sure how those are going to change with groups (if at all).  In any case, this comment might need to be updated when groups come along.